test/blackbox-tests/test-cases/watching/retriggering.t

```diff
@@ -8,39 +8,24 @@ Bad rule! You are not supposed to modify the source tree. No ice-cream for you!
   $ cat > dune <<EOF
   > (rule
   >   (deps (glob_files *.txt) (sandbox none))
-  >   (target result)
-  >   (action (bash "\| echo %{deps} > result
+  >   (alias default)
+  >   (action (bash "\| echo "I'm seeing: %{deps}"
   >                 "\| touch ../../new-source.txt
-  >                 "\| sleep 0.01
```

Comment:
Maybe we still need this `sleep`?

Reply:
Actually, never mind. The CI failure can't be explained by this rule finishing too quickly...

Reply:
Hmm, maybe the action gets killed too quickly and the action output is lost.

Reply:
Ah, that's it probably! That's the reason I was writing the result to a file and why I chose `sandbox none` in this test. I had exactly the same issue before.

Reply:
(So, just redirecting the output to `result` as before should fix this.)

Reply:
I changed the test to print in a file outside of the workspace rather than on stdout. This way we are sure the output won't be lost

Reply:
Let's see how the CI likes it

Reply:
Yeah, that should work too. Let's see!

Reply:
> (So, just redirecting the output to result as before should fix this.)

Indeed, that would work as well. I slightly prefer writing to a non-managed file like this as we can observe the two executions of the rule.

Reply:
Agreed!